build/dependencies.props

```diff
@@ -15,7 +15,8 @@
     <MicrosoftCodeAnalysisNetAnalyzersPackageVersion>7.0.0-preview1.22464.1</MicrosoftCodeAnalysisNetAnalyzersPackageVersion>
     <MicrosoftCrankEventSourcesPackageVersion>0.2.0-alpha.21255.1</MicrosoftCrankEventSourcesPackageVersion>
     <MicrosoftExtensionsLoggingTestingPackageVersion>2.1.1</MicrosoftExtensionsLoggingTestingPackageVersion>
-    <MicrosoftExtensionsPackageVersion>3.0.3</MicrosoftExtensionsPackageVersion>
+    <MicrosoftExtensionsPackageVersion>7.0.0-rc.2.22472.3</MicrosoftExtensionsPackageVersion>
+    <MicrosoftExtensions3PackageVersion>3.0.3</MicrosoftExtensions3PackageVersion>
```

Comment:
Can you at least use 3.1.x?

Reply:
I want to leave it as this for now, but I think soon we'll remove old targets from grpc-dotnet and upgrade this package version to something more modern.